samples/bluetooth/central_hids/README.rst

```diff
@@ -1,22 +1,132 @@
 .. _bluetooth_central_hids:
 
 Bluetooth: Central HIDS
-#######################################
+#######################
+
+The Central HIDS sample demonstrates how to use the :ref:`hids_c_readme` to interact with a HIDS server.
+Basically, the sample simulates a computer that connects to a mouse or a keyboard.
 
 Overview
 ********
 
-Bluetooth central application that connects to the HID devices.
+The sample scans available devices, searching for a HIDS server.
+If any HIDS server is found, the sample connets to it and discovers all characteristics.
+
+If any input reports are detected, the sample subscribes to them to receive notifications.
+If any boot reports are detected, the behavior depends on if they are boot mouse reports or boot keyboard reports:
+
+* If a boot mouse report is detected, the sample subscribes to it.
```

Comment:
Is that what you meant with "sign in"?

Reply:
Subscribe - But I can see you have already fixed it, right?

Reply:
Yes, I took a guess. ;)